crates/bws/src/config.rs

```diff
@@ -36,17 +38,23 @@ impl ProfileKey {
             ProfileKey::server_base => p.server_base = Some(value),
             ProfileKey::server_api => p.server_api = Some(value),
             ProfileKey::server_identity => p.server_identity = Some(value),
+            ProfileKey::state_file_path => p.state_file_path = Some(value),
         }
     }
 }
 
-pub(crate) const FILENAME: &str = "config";
-pub(crate) const DIRECTORY: &str = ".bws";
+pub(crate) const BWS_DIRECTORY: &str = ".bws";
+pub(crate) const STATE_DIRECTORY: &str = "state";
+pub(crate) const CONFIG_FILENAME: &str = "config";
+pub(crate) const STATE_FILENAME: &str = "state";
```

Comment:
This is more a suffix than anything.

Reply:
Resolved in [566545d](https://github.com/bitwarden/sdk/pull/388/commits/566545d475abb1fd04eb825090e4227df841f4ed)